scripts/mirror-images.sh

```diff
@@ -0,0 +1,25 @@
+set -e
+
+WORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+MANIFESTS_DIR=${WORK_DIR}/../manifests
+MANIFEST_TMP_DIR=${MANIFESTS_DIR}/tmp
+
+mirror_images() {
+    set -o errexit
+    failures=0
+    for mapping in ${MANIFESTS_DIR}/integreatly-*/image_mirror_mapping; do
+        echo "Running: oc image mirror -f=$mapping --skip-multiple-scopes $ARGS"
+        if ! oc image mirror -f="$mapping" --skip-multiple-scopes $ARGS; then
+            echo "ERROR: Failed to mirror images from $mapping"
+            failures=$((failures+1))
+        fi
+    done
+    exit $failures
+}
+
+# If you are running on mac you need to pass an additional arg
+# to the script --filter-by-os=.* We wont need this in the pipeline
+# as the os is linux.
+ARGS=$1
```

Comment:
You could optionally just set these args in the script if the OS is Darwin. Up to you, i will never need it :man_shrugging: